internal/fees/manager.go

```diff
@@ -81,11 +81,11 @@ func (f *Manager) ComputeNext(currTime int64, r Rules) (*Manager, error) {
 	minUnitPrice := r.GetMinUnitPrice()
 	lastTimeSeconds := int64(binary.BigEndian.Uint64(f.raw[0:consts.Int64Len]))
 	currTimeSeconds := currTime / consts.MillisecondsPerSecond
-	since := currTimeSeconds - lastTimeSeconds
+	since := uint64(currTimeSeconds - lastTimeSeconds)
```

Comment:
Can we add an error check here for an underflow?

Seems this would have previously triggered a panic by indexing into a negative index and we should already protect against this elsewhere, but better to either return an error or add a comment that clearly places the responsibility on the caller